test/libsolidity/syntaxTests/inheritance/base_arguments_empty_parentheses.sol

```diff
@@ -3,4 +3,5 @@ contract Base {
 }
 contract Derived is Base(2) { }
 contract Derived2 is Base(), Derived() { }
-contract Derived3 is Base, Derived {}
```

Comment:
Why remove this?

Reply:
``Derived2`` now is an error, whereas ``Derived3`` is still legal - a new test corresponding to the case of ``Derived3`` is in ``base_arguments_no_parentheses.sol``.

Reply:
`Derived3` is "legal" before the change. If it we consider it a bug, it can be made illegal in the non-breaking release, but it should be mentioned in the changelog. The current changelog change only mentions "empty parenthesis".

Reply:
The behaviour of the case of ``Derived3`` does not change - it will be "legal" after this change as well. But since after the change ``Derived2`` and ``Derived3`` of the original test case have different behaviour, the original test case is split into two and the case of ``Derived3`` is now in ``base_arguments_no_parentheses.sol`` (which still verifies that ``Derived3`` is "legal").

Reply:
Sorry, read it as "still illegal".

Can you add another test file then which has `Derived3`?

Reply:
=> ``base_arguments_no_parentheses.sol`` - it's just called ``Derived2`` and not ``Derived3`` there.